OpenRA.Mods.Common/UpdateRules/UpdatePath.cs

```diff
@@ -60,6 +60,7 @@ public class UpdatePath
 				new ModernizeDecorationTraits(),
 				new RenameHealCrateAction(),
 				new RenameInfiltrationNotifications(),
+				new RenameMPTraits(),
```

Comment:
This needs to be in a new `UpdatePath` (and the class in a new subdirectory) since we have now passed the prep branchoff for this previous one.